openstf/feature_engineering/general.py

```diff
@@ -76,3 +80,37 @@ def remove_extra_feature_columns(
         )
 
     return input_data.drop(extra_features, axis=1)
+
+
+def enforce_feature_order(df):
+    """Enforces correct order of features.
+
+    Alphabetically orders the feature columns. The load column remains the first column
```

Comment:
This is not completely inline with what the function does. I also think the general term used in Python and pandas is sort/sorting instead of orders (not even sure that is the right English term, google says orders is bestellingen (Dutch))

The missing information is:
* that you assume the first column contains the load
* And that the Horizon column (if present) is moved to the back in any case (also if it was somewhere else)

Reply:
Added this information